lib/evaluator.ex

```diff
@@ -1,25 +1,34 @@
 defmodule Veritaserum.Evaluator do
   @moduledoc """
-  Evaluats words, boosters, negators and emoticons.
+  Evaluates words, boosters, negators and emoticons.
   """
 
   ["word", "emoticon", "negator", "booster"]
   |> Enum.each(fn facet ->
-    File.read!("#{__DIR__}/../config/facets/#{facet}.json")
-    |> Jason.decode!()
-    |> (fn list ->
-          def unquote(:"#{facet}_list")(),
-            do: unquote(list |> Enum.map(fn {key, _} -> key end) |> List.flatten())
+    map =
```

Comment:
can we call this something more descriptive than `map`? I'm thinking `facet` or `facet_mapping`

Reply:
addressed